site/specs/numbers.json

```diff
@@ -2752,34 +2731,35 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves a list of disconnected numbers associated with the account. There are optional search parameters to limit the discNumbers payload</p>\n",
-        "operationId": "GET /accounts/{accountId}/discnumbers",
+        "description": "<p>Retrieves a list of disconnected numbers associated with the account. There are optional search parameters to limit the discNumbers payload</p>",
+        "operationId": "GetDisconnectedTns",
+        "summary": "Retrieves disconnected tns",
```

Comment:
Please change all summaries to the imperative form of the verb (`Retreive` instead of `Retrieves`)